app/uk/gov/hmrc/vulnerabilities/persistence/VulnerabilitiesTimelineRepository.scala

```diff
@@ -54,4 +63,44 @@ class VulnerabilitiesTimelineRepository @Inject()(
     )
     collection.bulkWrite(bulkWrites).toFuture().map(_ => ())
   }
+
+  val Quoted = """^\"(.*)\"$""".r
+
+  def getTimelineCountsForService(service: Option[String], team: Option[String], vulnerability: Option[String], from: Instant, to: Instant): Future[Seq[VulnerabilitiesTimelineCount]] = {
```

Comment:
AMBER: Wouldn't it make sense to filter for curation status here?

Reply:
resolved :) Logic is now in backend rather than frontend